app/src/main/java/org/p2p/wallet/history/ui/history/HistoryPresenter.kt

```diff
@@ -110,6 +111,7 @@ class HistoryPresenter(
 
     private suspend fun fetchMoonpayTransactions() {
         val transactions = sellInteractor.loadUserSellTransactions()
+            .filter { transaction -> transaction.isCancelled() }
```

Comment:
Did you mean `filterNot`? 
You need all transactions except `cancelled` right?

Reply:
Yes, my mistake. replace `transaction -> !transaction.isCancelled()`